src/main/scala/EnsimePlugin.scala

```diff
@@ -474,7 +474,8 @@ object EnsimePlugin extends AutoPlugin {
 
       val target = targetForOpt(config).get
       val scalaCompilerArgs = ((scalacOptions in config).run ++
-        ensimeSuggestedScalacOptions(ensimeScalaV)).toList
+        (ensimeScalacOptions in config).run ++
+        ensimeSuggestedScalacOptions(ensimeScalaV)).toList.distinct
```

Comment:
`.distinct` can break things unexpectedly... e.g. consider two flags that take `utf8` with space separator. I seem to recall that being a real problem before. Whereas `scalac` is fine with duplicate flags.